pkg/plugins/manager/loader/angulardetector/service.go

```diff
@@ -12,16 +12,14 @@ import (
 // They are executed in the specified order.
 var defaultDetectors = []detector{
 	&containsBytesDetector{pattern: []byte("PanelCtrl")},
-	&containsBytesDetector{pattern: []byte("QueryCtrl")},
+	&containsBytesDetector{pattern: []byte("ConfigCtrl")},
 	&containsBytesDetector{pattern: []byte("app/plugins/sdk")},
 	&containsBytesDetector{pattern: []byte("angular.isNumber(")},
 	&containsBytesDetector{pattern: []byte("editor.html")},
 	&containsBytesDetector{pattern: []byte("ctrl.annotation")},
 	&containsBytesDetector{pattern: []byte("getLegacyAngularInjector")},
 
-	&regexDetector{regex: regexp.MustCompile(`['"](app/core/utils/promiseToDigest)|(app/plugins/.*?)|(app/core/core_module)['"]`)},
-	&regexDetector{regex: regexp.MustCompile(`from\s+['"]grafana\/app\/`)},
-	&regexDetector{regex: regexp.MustCompile(`System\.register\(`)},
+	&regexDetector{regex: regexp.MustCompile(`["']QueryCtrl["']`)},
```

Comment:
This fixes the ML app being detected as Angular:
https://github.com/grafana/machine-learning/blob/d8b1cf78332c3fba26abab3118e0eaa338dc0a4b/ui-plugins/grafana-ml-common/components/QueryEditor/AngularQueryEditor.tsx#L94